rad/src/lib.rs

```diff
@@ -208,6 +215,12 @@ pub async fn run_retrieval_report(
 
             result
         }
+        RADType::Rng => {
+            let random_bytes: [u8; 32] = rand::random();
+            let random_bytes = RadonTypes::from(RadonBytes::from(random_bytes.to_vec()));
+
+            Ok(RadonReport::from_result(Ok(random_bytes), context))
```

Comment:
This ignores the retrieval script, according to the WIP the script should be executed.

Reply:
Well actually I'm not sure if the retrieval script should be executed or not.

Reply:
Let's discuss here:
https://github.com/witnet/WIPs/pull/70#discussion_r706076131

Reply:
What was the conclusion?